__tests__/2-GET-api-topics.test.js

```diff
@@ -0,0 +1,32 @@
+const request = require("supertest");
+const app = require('../app');
+const { topicData } = require("../db/data/test-data");
+const db = require(`../db/connection`);
+const seed = require(`../db/seeds/seed`);
+const data = require('../db/data/test-data')
+
+beforeEach(() => seed(data));
+afterAll(() => db.end());
+
+describe('GET /api/topics',() => {
+    test('GET:200 availble on endpoint /api/topics', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then((response) =>  {
+            const body = response.body
```

Comment:
As we are using a forEach here - if the array is empty then this test will pass by default as none of the expect statements will be called. A test with no expect statements will pass by default. We should first check that there is something in the array (check its length)